pilot/tools/debug/pilot_cli.go

```diff
@@ -115,10 +116,16 @@ func (p *PodInfo) populatePodNameAndIP(kubeconfig string) bool {
 }
 
 func (p PodInfo) makeNodeID() string {
+	if p.NodeType != "" {
```

Comment:
You can remove this, if node type actually "sidecar" by default.

Reply:
done.